src/Mvc/Mvc.NewtonsoftJson/src/NewtonsoftJsonInputFormatter.cs

```diff
@@ -225,7 +225,8 @@ void ErrorHandler(object sender, Newtonsoft.Json.Serialization.ErrorEventArgs ev
                         }
                         else
                         {
-                            addMember = !path.EndsWith("." + member, StringComparison.Ordinal);
+                            addMember = !path.EndsWith($".{member}", StringComparison.Ordinal)
```

Comment:
Interesting. Did we miss this scenario with json.net?

Reply:
The JSON path logic in S.T.J is slightly different from Json.NET

Reply:
btw I know this isn't a hot path but string interpolation instead of concatenating two strings together is so wasteful.